### ISSUE TITLE
Netbird sets itself as default dns resolution

### ISSUE THREAD

**Issue by apika** [netbirdio/netbird#988]
**Describe the problem**
When trying to resolve a DNS query to a machine on another VPN only by name (without the domain), the domain used is the one set on netbird.

**To Reproduce**
1. Connect to a VPN that has a DNS server configured.
2. Connect to a Netbird VPN.
3. Ping to a machine on the other VPN, without using the domain

**Expected behavior**
Netbird domain should be the last one used for domain resolution (or, at least, have an option to be) to avoid beaking DNS querys to other domains on different networks.

**Additional context**
Detected while using Wireguard or OpenVPN while using Netbird at the same time.

**Comment by mlsmaycon** [comment]
Hello @apika just to confirm the behavior, this happens when you have an all-domain resolver set in the dashboard, right? In other words, you are not limiting the domains that netbird is responsible to respond to?

**Comment by apika** [comment]
No, i wasn't limiting the domains that netbird should respond. Sadly I don't have netbird anymore so I can't expand much more on this issue.

Something that I forgot to add to the issue is that the computer was a joined to a Windows AD and the domain that I couldn't reach was the one from de AD.

**Comment by MobileManiC** [comment]
Same issue on Windows I would say... In a lot of cases AD DNS suffix is needed to work properly, otherwise local services will not work at all... (This seens to include AD as well.)

It's OK to add netbird.cloud search domain, but original computer defined domain should be recognized as well....

**Comment by nazarewk** [comment]
I believe this was addressed in one of recent releases, feel free to comment if it's still relevant for you and we'll reopen.
